src/components/concentration-display/Concentration.tsx

```diff
@@ -36,7 +38,7 @@ const Concentration: React.FC<AgentProps> = ({
         agent: AgentName,
         currentConcentrationOfAgent: number
     ) => {
-        if (adjustableAgent === agent && !isPlaying && page !== 2) {
+        if (adjustableAgent === agent && !isPlaying && page > 3) {
```

Comment:
Is there any place in this repo where the page number is mapped to some kind of name? As someone looking at this for the first time, the page numbers don't really mean anything to me. I'm also curious as to what the plan is if the team decides to add an extra page / step between existing pages.